101-media-services-create/azuredeploy.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "name": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Media Services Account."
+      }
+    },
+    "storageName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Storage."
+      }
+    },
+    "storageType": {
```

Comment:
to minimize input and maintenance - use a literal value on the resource and pick a default that will work across clouds (mooncake, gov, etc)